examples/window/window_drag_move.rs

```diff
@@ -0,0 +1,140 @@
+        .insert_resource(LeftClickAction::Drag)
+        .add_systems(Startup, setup)
+        .add_systems(Update, (handle_input, move_windows))
+        .run();
+}
+
+fn setup(mut commands: Commands) {
+    // Camera
+    commands.spawn(Camera3d::default());
+
+    // UI
+    commands
+        .spawn((
+            NodeBundle {
+                style: Style {
+                    position_type: PositionType::Absolute,
+                    padding: UiRect::all(Val::Px(5.0)),
+                    ..default()
+                },
+                background_color: Color::BLACK.with_alpha(0.75).into(),
+                ..default()
+            },
+            GlobalZIndex(i32::MAX),
+        ))
+        .with_children(|p| {
+            p.spawn(Text::default()).with_children(|p| {
+                p.spawn(TextSpan::new(
+                    "Demonstrate drag move and drag resize without window decorations.\n\n",
+                ));
+                p.spawn(TextSpan::new("Controls:\n"));
+                p.spawn(TextSpan::new("A - change left click action ["));
+                p.spawn(TextSpan::new("Drag"));
+                p.spawn(TextSpan::new("]\n"));
+                p.spawn(TextSpan::new("S / D - change resize direction ["));
+                p.spawn(TextSpan::new("NorthWest"));
+                p.spawn(TextSpan::new("]\n"));
+            });
+        });
+}
+
+fn handle_input(
+    input: Res<ButtonInput<KeyCode>>,
+    mut action: ResMut<LeftClickAction>,
+    mut dir: ResMut<ResizeDir>,
+    example_text: Query<Entity, With<Text>>,
+    mut writer: UiTextWriter,
+) {
+    use LeftClickAction::*;
+    if input.just_pressed(KeyCode::KeyA) {
+        *action = match *action {
+            Drag => Resize,
+            Resize => Nothing,
+            Nothing => Drag,
+        };
+        *writer.text(example_text.single(), 4) = format!("{:?}", *action);
+    }
+
+    if input.just_pressed(KeyCode::KeyS) {
+        dir.0 = dir
+            .0
+            .checked_sub(1)
+            .unwrap_or(DIRECTIONS.len().saturating_sub(1));
+        *writer.text(example_text.single(), 7) = format!("{:?}", DIRECTIONS[dir.0]);
+    }
+
+    if input.just_pressed(KeyCode::KeyD) {
+        dir.0 = (dir.0 + 1) % DIRECTIONS.len();
+        *writer.text(example_text.single(), 7) = format!("{:?}", DIRECTIONS[dir.0]);
+    }
+}
+
+fn move_windows(
+    mut windows: Query<&mut Window>,
+    action: Res<LeftClickAction>,
+    input: Res<ButtonInput<MouseButton>>,
+    dir: Res<ResizeDir>,
+) {
+    // Both `start_drag_move()` and `start_drag_resize()` must be called after a
+    // left mouse button press as done here.
+    //
+    // winit 0.30.5 may panic when initiated without a left mouse button press.
+    if input.just_pressed(MouseButton::Left) {
+        for mut window in windows.iter_mut() {
+            match *action {
+                LeftClickAction::Nothing => (),
+                LeftClickAction::Drag => window.start_drag_move(),
```

Comment:
```suggestion
                LeftClickAction::Move => window.start_drag_move(),
```
this would be more consistent with `drag_move` vs `drag_resize`